aws-lc-rs/src/kem.rs

```diff
@@ -12,7 +12,7 @@
 //! ```ignore
```

Comment:
Also un-ignore this now that a feature isn't required for this. This will allow this to be tested. There was a duplicate example in the unstable module if you want to remove it all together over there.